config.ini

```diff
@@ -1,8 +0,0 @@
-[Frame]
-width=80
-height=20 
-
-[General]
-independent_graphs=False
-timezone=America/New_York
-rounding_mode=math
```

Comment:
this was accidental due to the ignores but this is no longer needed due to the argument defaults made in `def merge_config(config, args):`

Reply:
I would still love the existence of at least this config file - not having a portfolio.ini makes sense (it's such a specific file for each user). This one stands as a basis for users to look at config syntax, and having it in the repo by default makes it really easy to change just one setting.
Glad we got rid of gen_config(), but I would still like this file to exist - even if it is redundant.

Reply:
re-added and other comments addressed in #30